language/message/fr_FR.po

```diff
@@ -929,12 +932,12 @@ msgid "Back to favorite"
 msgstr "Retour aux favoris"
 
 #: public/build/javascript/bundle.js:2556
-msgid "FAVORIS"
-msgstr ""
+msgid "FAVORITES"
+msgstr "FAVORIS"
```

Comment:
The translation key is to change here.  
This change needs to be updated in `poi_panel.dot` and to be synchronized with the other languages.